packages/server/src/common/launch/glsp-server-launcher.ts

```diff
@@ -19,10 +19,8 @@ import { ServerModule } from '../di/server-module';
 import { InjectionContainer } from '../di/service-identifiers';
 import { Logger } from '../utils/logger';
 
-export const START_UP_COMPLETE_MSG = '[GLSP-Server]:Startup completed';
-
 @injectable()
-export abstract class GLSPServerLauncher<T = undefined> {
+export abstract class GLSPServerLauncher<T> {
```

Comment:
Should we provide a `protected disposeOnStop = new DisposableCollection();` to provide a default `stop` behavior that can be filled up by adding disposables to this collection on `start` or `run`? Maybe this launcher should also implement `Disposable`.

Reply:
Good idea 👍🏼